src/ops/ops.go

```diff
@@ -181,6 +181,15 @@ func (o *ops) SetBootOrder(device string) error {
 		return nil
 	}
 
+	o.log.Infof("SetBootOrder, runtime.GOARCH: %s, device: %s", runtime.GOARCH, device)
```

Comment:
This change should come with a matching unit test. Where is it? :smiley:

Reply:
Right, that's exactly the problem. If someone in the future removes those lines you're adding here (or does any other change) this PR is going to get merged without anything to stop it
Can you please add tests?